src/SmartComponents/BaselinesPage/EditBaseline/EditBaseline.js

```diff
@@ -265,6 +276,8 @@ export class EditBaseline extends Component {
 
         return (
             <Checkbox
+                data-ouia-component-type='PF4/Checkbox'
+                data-ouia-component-id={ 'checkbox-' + fact[1] }
```

Comment:
`data-ouia-component-id={ 'checkbox-' + fact[FACT_NAME] }`
`FACT_ID`, `FACT_NAME` and `FACT_VALUE` are constants that are saved in another file. They are 0, 1 and 2, respectively. So instead of using 1 here, you can just use `FACT_NAME`.